doc/source/cluster/deploy.rst

```diff
@@ -13,6 +13,7 @@ Ray with Cluster Managers
    :maxdepth: 2
 
    kubernetes.rst
+   kuberay.md
```

Comment:
@pcmoritz Actually any reason we are using .md for this one vs. .rst?

Reply:
maybe because .rst is awful :)

For consistency, we should probably switch back to .rst as we solidify the integration and its documentation.